lib/model/CTokenListCategory.cc

```diff
@@ -309,13 +309,15 @@ bool CTokenListCategory::updateOrderedCommonTokenIds(const TSizeSizePrVec& newTo
 
     // Iterate over the possible starting positions within the current ordered
     // tokens.
+    std::size_t bestWeight{0};
     for (std::size_t tryOrderedCommonTokenBeginIndex = m_OrderedCommonTokenBeginIndex;
          tryOrderedCommonTokenBeginIndex < m_OrderedCommonTokenEndIndex;
          ++tryOrderedCommonTokenBeginIndex) {
 
         std::size_t newIndex{0};
-        for (std::size_t commonIndex = tryOrderedCommonTokenBeginIndex;
-             commonIndex < m_OrderedCommonTokenEndIndex; ++commonIndex) {
+        std::size_t tryWeight{0};
+        std::size_t commonIndex = tryOrderedCommonTokenBeginIndex;
+        for (; commonIndex < m_OrderedCommonTokenEndIndex; ++commonIndex) {
```

Comment:
I think `commonIndex` can still be initialized in the `for` loop?  It doesn't seem to be used outside the loop.